pytransifex/api.py

```diff
@@ -144,20 +148,20 @@ def create_resource(
             logging.info(f"Resource created: {resource_slug or resource_name}")
 
         else:
-            raise Exception(
+            raise ValueError(
                 f"Not project could be found wiht the slug '{project_slug}'. Please create a project first."
             )
 
     @ensure_login
     def update_source_translation(
-        self, project_slug: str, resource_slug: str, path_to_file: Path
+        self, *, project_slug: str, resource_slug: str, path_to_file: str
```

Comment:
**Potential compatibility issue**: This function now disallows posittional arguments.